File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int l = 0;
        int r = 0;
        int len= nums.length;
        while(r < len) {
            nums[l] = nums[r];
            while(r < len && nums[r] == nums[l]) {
                r++;
            }
            l++;
        }

        return l;
        
    }

    
}

File: Data Structures & Algorithms/sort-colors/submission-1.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        colors = [0, 0, 0]
        for num in nums:
            colors[num] += 1
        k = 0
        for i in range(len(colors)):
            for j in range(colors[i]):
                nums[k] = i
                k+=1
        
        